App.java

```diff
@@ -0,0 +1,89 @@
+
+//import java.io.Console;
+import java.io.*;
+
+public class App {
+    public static void main(String[] args) throws Exception {
+
+        System.out.println("Starting Program!");
+        // Day 1 - Part 1
+        try {
+            File file = new File("/Users/dpucci/Java/HelloWorld/day1input.txt");
+            int counter = 0;
+            if (file.exists() && !file.isDirectory()) {
+                FileReader fr = new FileReader(file); // reads the file
+                BufferedReader br = new BufferedReader(fr); // creates a buffering character input stream
+                String line;
+                int number1 = 0;
+                int number2 = 0;
+                // set first line
+                if ((line = br.readLine()) != null) {
+                    number1 = Integer.parseInt(line);
+                }
+                while ((line = br.readLine()) != null) {
+                    number2 = Integer.parseInt(line);
```

Comment:
Is the intent of this code to count how many a number below is larger than the number on the line above?

Reply:
Yes